add-file-headers.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+#
+# Helper script to add headers to source files.
+# Invoke without arguments to display usage.
+#
+# date:   2018-19-01
+# author: Mat Hansen <mhansen@eclipsesource.com>
+#
+
+MODE="undefined"
+HEADER_FILE=""
```

Comment:
Is there no switch to set `HEADER_FILE`? Otherwise it should be hard-coded to `LICENSE`, I guess?

Reply:
In fact there is one (-h=*|--header=*).  But you are right, it is missing in the usage example. I will take care of that.